packages/branding/package.json

```diff
@@ -1,5 +1,11 @@
 {
   "name": "@taiko/branding",
   "version": "0.1.1",
-  "private": true
+  "private": true,
+  "devDependencies": {
+    "tailwindcss": "^3.3.2"
+  },
+  "dependencies": {
+    "daisyui": "1.16.6"
+  }
```

Comment:
do we need these now? if it's just for later, i feel like we can not include it now and include it if/when we need it. so just the `tailwind.config.js` could be sufficient to keep track of now, and we will update that `tailwind.config.js` with the correct configuration as per the new design and use it where we need it.

but if we do include it, `daisyUI` can be moved to`devDependencies`

Reply:
bump @jscriptcoder

Reply:
bumped... regarding whether we need or not the dependencies, at the moment we don't. DaisyUI should be though a peerDependency, since it's getting imported in tailwind.config.js, and if we don't want the IDE to complain we should add it... I'll remove them